src/partials/hero.html

```diff
@@ -0,0 +1,22 @@
+<section class="hero-background">
+    <div class="hero-container">
+        <h1 class="visually-hidden">Simply chocolate - SHOP</h1>
+        <h2 class="hero-text">Treat yourself or a loved one to<br/> our finest ingredients for a moment<br/> of pure delight!</h2>
```

Comment:
osobiście nie wiem, czy br jest tutaj odpowiedni, patrząc na responsywność. Bo jak zobaczysz na figmie w mobile w innym miejscu ścina się nagłówek. Więc może spróbuj wielkość zdefiniować w css, zamiast sztywno br'ami

Reply:
To <br/> wiem, ale bez tego z kolei mobile jest ok, a desktop i tablet pokazuje inaczej, bo krótkie wyrazy przeskakują do góry, bo próbowałam tak i tak. Może jeszcze pokombinuję

Reply:
Wiesz co, połączyłam jedno i drugie.. Czyli zostawiłam break po "to" tylko i ustawiłam szerokość elementu dla tabletu i desktopa. Więc our przeskakuje do nast linijki, a reszta ustawia się pod szerokość. Pokazuje już ok, może przejdzie ;d